src/allencell_ml_segmenter/core/prediction_result_input_widget.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Optional
+
+from allencell_ml_segmenter.core.file_input_model import (
+    InputMode,
+    FileInputModel,
+)
+from allencell_ml_segmenter.core.file_input_widget import FileInputWidget
+from allencell_ml_segmenter.main.i_viewer import IViewer
+from allencell_ml_segmenter.main.segmenter_layer import LabelsLayer
+from qtpy.QtCore import Qt
+
+from allencell_ml_segmenter.prediction.service import ModelFileService
+
+from napari.utils.events import Event as NapariEvent  # type: ignore
+
+
+class PredictionResultListWidget(FileInputWidget):
+    """
+    Widget containing a list of prediction results that are selectable for thresholding
+    """
+
+    def __init__(
+        self, model: FileInputModel, viewer: IViewer, service: ModelFileService
+    ):
+        super().__init__(
+            model, viewer, service, include_channel_selection=False
+        )
+        self._prediction_layers: list[LabelsLayer] = []
+
+    def _update_layer_list(self, event: Optional[NapariEvent] = None) -> None:
+        previous_selections: list[int] = self._image_list.get_checked_rows()
+        self._image_list.clear()
+        self._prediction_layers = (
+            self._viewer.get_all_layers_containing_prob_map()
+        )
+        for idx, prediction_output_layer in enumerate(self._prediction_layers):
+            self._image_list.add_item(
+                prediction_output_layer.name,
+                set_checked=idx in previous_selections,
+            )
+
+    def _process_checked_signal(self, row: int, state: Qt.CheckState) -> None:
```

Comment:
_update_layer_list and _process_checked_signal appear to be unused (they are new code, and i dont see them referenced by other new code in PredictionResultListWidget)?